Week-1/tanmay-sinha/url_extracter.py

```diff
@@ -0,0 +1,39 @@
+import time
```

Comment:
- Few Bad-Whitespace issues.
- Module Docstring None.
- Constants: UPPER_CASE.
- Single character variables are just not helpful.